### PR TITLE
Landing page rework

### PR DESCRIPTION
**Pull request by adamjhc**
- Rewrote css in linaria
- Moved svgs into the assets folder
- Removed tab bar from landing page

### CODE REVIEW

File: src/pages/landing/index.tsx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  `
+
+  const List = styled.ul`
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    margin-top: var(--space-xs);
+
+    li::before {
+      content: "👉 ";
```
Comment (pavsidhu) [inline src/pages/landing/index.tsx:206]: Niceee 👌🏼

**Review by pavsidhu — CHANGES_REQUESTED** [review]

File: src/pages/landing/index.tsx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+      text-align: center;
+      font-weight: bold;
+    }
+  `
+
+  return (
+    <Container>
+      <FeelingLogo />
+      <Links>
+        <Link href="https://twitter.com/feelingapp">
```
[the remark this answers is not in the document]
Comment (adamjhc) [reply src/pages/landing/index.tsx:54]: sounds good to me

File: src/pages/landing/index.tsx [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+      </Links>
+    </Container>
+  )
+}
+
+function Main() {
+  const Container = styled.main`
+    --grid-width: calc(100vw - var(--space-l));
+
+    width: var(--grid-width);
```
[the remark this answers is not in the document]
Comment (adamjhc) [reply src/pages/landing/index.tsx:71]: yeah I realised while I was doing this it was the same as padding, mainly kept it bc it's what I had before and I set the width in the media queries

File: src/pages/landing/index.tsx [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+      margin: 0 auto;
+      padding-top: var(--space-m);
+
+      grid-template-columns: repeat(2, 1fr);
+      grid-template-rows: repeat(3, 1fr);
+      grid-template-areas:
+        "title title"
+        "input input"
+        "description image";
+      /* gap: 0; */
```
[the remark this answers is not in the document]
Comment (adamjhc) [reply src/pages/landing/index.tsx:99]: will do

File: src/pages/landing/index.tsx [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+    }
+
+    @media (min-width: 1200px) {
+      font-size: 6rem;
+      line-height: 7rem;
+    }
+  `
+
+  const Input = styled.div`
+    --input-border-radius: 12px;
```
[the remark this answers is not in the document]
Comment (adamjhc) [reply src/pages/landing/index.tsx:131]: with a variety of sizes or just the one?

File: src/pages/landing/index.tsx [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+    --input-padding: 15px;
```
[the remark this answers is not in the document]
Comment (adamjhc) [reply src/pages/landing/index.tsx:132]: yup think I already replaced them elsewhere actually, nice catch

File: src/pages/landing/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (pavsidhu) [reply src/pages/landing/index.tsx:131]: nahh just --border-radius: 12px should be good 👍🏼

File: src/pages/landing/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamjhc) [reply src/pages/landing/index.tsx]: Decided to leave this as-is for now because the grid-width property works with a lot of the other elements

**Pull request merged**